content/pl/wydarzenia/pikniki.md

```diff
@@ -1,8 +1,14 @@
 ---
 title: Pikniki Hackerspace Trójmiasto
 tags: ["hs3", "piknik"]
+openingHours: >
```

Comment:
A comment here would work wonders, as, unfortunately, this is only slightly more readable than regex to most of the people

Reply:
OK, new, custom, format, that is parsable in Hugo, and readable by humans :)

```
eventDates:
    periodics:
    - start: 2022-10-01
      end: 2024-01-02
      starttime: 18:00
      endtime: 02:00
      rule: last Friday of the month
    - start: 2022-12-01
      end: 2023-02-10
      rule: every 14 days
      starttime: 12:00
      endtime: 14:00
      comment: Winter lunch meeting
    cancelled:
      2023-02-24: "Too cold"
      2023-07-28: vacation
    extra:
      2022-12-07 18:00-19:00: extra meeting too
      2023-08-27 18:00-19:00: extra meeting
      2022-10-29 18:00-05:00: DLS change party!
    changes:
      2022-10-28->2022-10-27: "moved because of planning issues"
      2022-12-30: bring 🍾
      2023-04-28->19:00-01:00: "starting a bit later"
      2023-10-27->2023-10-20 19:00-21:00: moved a week earlier

```

Also means we can kill the extra JS dependency (which is always a good idea).

And as 🍒 on the 🥧: it can generate an `.ics` from the page -- and you can even subscribe in your own calendar software, and see all changes being updated semi-real-time.

Should also be able to generate a huge HS-wide `.ics` to subscribe to )